tests/unit/brevo.t

```diff
@@ -10,6 +9,8 @@ use ProductOpener::APITest qw/:all/;
 use ProductOpener::Test qw/:all/;
 use File::Temp ();
 
+no warnings qw(experimental::signatures);
```

Comment:
Should we put it in all files ? Can't we put it directly in `ProductOpener::PerlStandards` ?